os_net_config/tests/test_cli.py

```diff
@@ -210,6 +210,12 @@ def apply(self, cleanup=False, activate=True):
                                            % interface_yaml, exitcodes=(0,))
 
     def test_sriov_noop_output(self):
+        def test_update_sriov_pf_map(name, numvfs, noop, promisc=None,
```

Comment:
I see that the method "test_update_sriov_pf_map" is defined in many places and then stubbed out.

I think we could move this to L49 and then only define it once and stub it out once. Do you agree?